Gemfile

```
@@ -119,9 +119,7 @@ group :development, :docker_development do
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "rubocop", "~> 1.50", require: false
-  gem "rubocop-packaging", require: false
-  gem "rubocop-performance", require: false
-  gem "rubocop-rails", require: false
+  gem "rubocop-shopify", require: false
```

Use a version specifier here. Otherwise, we might suddenly have breaking changes.

```suggestion
  gem "rubocop-shopify", "~> 2.14", require: false
```

---

Note that this will also update `rubocop` from `1.50.2` to `1.52.1`.

---

I think this is a good time to get some versions right. Let's update rubocop and rubocop-shopify together:

```rb
gem "rubocop", "~> 1.57", require: false
gem "rubocop-shopify", "~> 2.14", require: false
```

However, `bundle install` will then fail due to `erb_lint`. Let's deal with that in a future PR. For now, we just want to get Rubocop to work. Therefore, remove the line

```rb
gem "erb_lint", require: false
```

Then running `bundle install` should work just fine.

---

Done. Note that updating the rubocop version also triggered some updates of rubocop dependencies, see `Gemfile.lock`.